test/parallel/test-tls-js-stream.js

```diff
@@ -53,23 +48,15 @@ const server = tls.createServer({
   const socket = tls.connect({
     socket: p,
     rejectUnauthorized: false
-  }, function() {
+  }, common.mustCall(function() {
     console.log('client secure');
 
-    connected.client++;
-
-    socket.end('hello');
     socket.resume();
-    socket.destroy();
```

Comment:
For reference this was added in https://github.com/nodejs/node/commit/75930bb38c734fa0f8f4dbedb4187ce61dd08e3d#diff-94e518e66d7d1b91e52af17401305431.

Reply:
It's placement was a deliberate regression test for destroy() in an SSL cb (specifically, from js code executed during a callback from an SSL C++ cb), so removing it loses the regression test. But, the general purpose of this test for js streams, not destroy-in-cb, and it looks like the two tests don't fit comfortably together anymore. It would be good to have a new test that triggers the old bug, but maybe at this point, with the bug fixed, we can afford to lose this line. If not, we likely need a new test specifcally for the destroy that doesn't depend on the rest of the test working.